src/votekit/outcome_model.py

```diff
@@ -0,0 +1,140 @@
+from pydantic import BaseModel
+from profile import PreferenceProfile
+from typing import Optional
+
+#1. "--> int" arrow thing for each function
+#2. is add_winners_and_losers necessary? --A: probabably not
+#3. do we want anything more?]
+#4. RCV cruncher: inspiration for funcitons--> statistics functions/things
+# Example of immutable data model for results
+
+class Outcome():
```

Comment:
this should inherit from pydantic.BaseModel if that's the intention, later on pydantic Config is used, but inheritance is missing